src/components/pages/observation/list/views/shared/filterDownLoadModal.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  useDisclosure,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  Box
+} from "@chakra-ui/core";
+import CheckboxGroup from "@components/form/checkboxGroup";
+import { observationFilterList } from "@static/observationFiltersList";
+import TextBoxField from "@components/form/text";
+import Submit from "@components/form/submit-button";
+import useTranslation from "@configs/i18n/useTranslation";
+import { useStoreState } from "easy-peasy";
+import { axObservationFilterDownload } from "@services/observation.service";
+import notification, { NotificationType } from "@utils/notification";
+import * as Yup from "yup";
+
+export default function DownLoadFilterModal({ traits, customFields }) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { id } = useStoreState((s) => s.user);
+  const { t } = useTranslation();
+  const hform = useForm({
```

Comment:
Hook form is just an overkill here, plus if you are not going to use fields just merge it when submitting request